packages/okta-react/README.md

```diff
@@ -173,6 +173,7 @@ Security is the top-most component of okta-react. This is where most of the conf
 * **client_id** (required) - The OpenId Connect `client_id`
 * **redirect_uri** (required) - Where the callback handler is hosted
 * **onAuthRequired** (optional)
+* **auth** (optional) - Provide an Auth object instead of the options above
```

Comment:
Let's add a note about why this is useful, i.e. when you need to share the auth instance w/ other code, e.g. code that needs to get the access token

Reply:
it would be _really_ nice to see a simple example....like in your API.js service to get your token.  I just spent two days figuring it out